docs/building_bundled_executables.rst

```diff
@@ -47,10 +47,25 @@ software installed.
 Using the build script
 ======================
 
+.. note::
+
+    The build script creates a new virtual environment to build the bundled
+    executables, and so it should **not** be run from within an existing
+    virtual environment. Run the ``deactivate`` command if you are currently
+    in a virtual environment. You may also need to specify the full path to
+    the ``pycbc_build_eah.sh`` script once you leave your virtual environment.
+
 The command-line arguments for the ``pycbc_build_eah.sh`` build script are:
 
 .. command-output:: pycbc_build_eah.sh --force-debian4 --help
 
+The build script creates the directories ``pycbc-sources`` and ``pycbc-build``
+in the directory that it is run from.  The build script creates the bundles in
+the directory ``pycbc-build/environment/dist/`` relative to the directory from
+which it run. The ``pycbc_inspiral`` executable will be named
+``pycbc_inspiral_osg`` or ``pycbc_inspiral_osg_TAG`` if the git commit matches
+a tag name ``TAG`` in the repository.
```

Comment:
Actually not for an arbitrary tag, but only if the tag has the format of a revision number `^\.v[0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*$'`